### DISCUSSION TITLE
Quickstart guide

### DISCUSSION THREAD

**Discussion by jarvis-lorry** [siliconcompiler/siliconcompiler#856] [Help (Q&A)] [answered]
Hi, I'm setting up SiliconCompiler and am having trouble getting the heartbeat example in the quickstart guide to work because it seems sc cannot find the PDK library.  Am I supposed to load build PDK library onto my machine and tell SC how to find it.  How does target() find the PDK information?

Error from run:
ERROR ../third_party/pdks/virtual/freepdk45/libs/NangateOpenCellLibrary/r1p0/lib/NangateOpenCellLibrary_typical.lib ws not found.

Thank you for any help.  I'm new here.

**Comment by nmoroze** [comment, marked as answer]
Hi! For local runs, you'll need to clone the SC repository and then set the environment variable `SCPATH` to `<path-to-sc-repo>/siliconcompiler`, and that should do the trick! 

Sorry this isn't clarified in the quickstart guide, I can look into adding a note to make this requirement clear (note: this shouldn't be necessary if you use the remote, because the PDKs are installed on our server).

**Comment by jarvis-lorry** [comment]
Setting SCPATH to <path-to-sc-repo>/siliconcompiler has helped me but now I'm having difficulty installing OpenROAD.  Is OpenROAD supposed to be installed and built separately from SiliconCompiler?  Am I supposed to install OpenROAD within the virtual Python environment?  Does it matter?

**Reply by jarvis-lorry** [reply]
Hi, I've been able to install OpenROAD and have gotten further in running Silicon Compiler on the heartbeat design but SC errors out during global route.  Is there anything obvious that I'm doing wrong?

[ERROR STA-0406] global_route positional arguments not supported.
Error: sc_route.tcl, 37 STA-0406

**Reply by jarvis-lorry** [reply]
I've resolved the issue by rebuilding OpenROAD with an earlier commit labelled d1c7001ad, which was suggested on this discussion board.

**Reply by nmoroze** [reply]
Sorry for not getting back to this earlier - but I'm glad you were able to resolve this and thanks for following up with your solution! Were you able to get fully through the rest of the quickstart guide without any other issues?

**Reply by jarvis-lorry** [reply]
Yes, I was able to run the complete flow for the heartbeat example.
